maproulette/src/main/java/com/pixeltron/mapquest/open/geocoding/GeocodingRequest.java

```diff
@@ -5,27 +5,28 @@
 
 import org.apache.commons.lang3.StringUtils;
 
-public class GeocodingRequest {
+public class GeocodingRequest implements Request {
 	
 	private String API_KEY = "Fmjtd%7Cluubnu0zl9%2Caa%3Do5-9u1g0y";
 	private String BASE_URL = "http://open.mapquestapi.com/geocoding/v1/batch";
-	
-	public String buildUrl(String location1, String location2, Boolean thumbMaps) throws UnsupportedEncodingException {
+	private Boolean THUMB_MAPS = false;
+
+	public String buildUrl(RequestParameters parameters) throws UnsupportedEncodingException {
 		StringBuilder urlBuilder = new StringBuilder(BASE_URL);
 		urlBuilder.append("?key=").append(API_KEY);
 		urlBuilder.append("&inFormat=kvp&outFormat=json");
-		if (StringUtils.isNotBlank(location1)) {
-			urlBuilder.append("&location=").append(URLEncoder.encode(location1, "UTF-8"));
+		if (StringUtils.isNotBlank(parameters.start)) {
+			urlBuilder.append("&location=").append(URLEncoder.encode(parameters.start, "UTF-8"));
 		} else {
 			return null;
 		}
-		if (StringUtils.isNotBlank(location2)) {
-			urlBuilder.append("&location=").append(URLEncoder.encode(location2, "UTF-8"));
+		if (StringUtils.isNotBlank(parameters.end)) {
+			urlBuilder.append("&location=").append(URLEncoder.encode(parameters.end, "UTF-8"));
 		} else {
 			return null;
 		}
-		if (thumbMaps != null) {
-			urlBuilder.append("&thumbMaps=").append(thumbMaps);
+		if (THUMB_MAPS != null) {
```

Comment:
This will never be null with the given code changes - I would either update it to read off of parameters (eg `parameters.thumbMaps` or remove the null check. Reading off of `parameters` would be more idiomatic with the rest of the changes.